### ISSUE TITLE
Issue with Next up and collection

### ISSUE THREAD

**Issue by KCRN33** [nixgates/plugin.video.seren#689]
I am using seren 2.1.4 from nixgates repo 2.1.0 on Kodi 19.1 on a Nvidia Shield device. I am having issues with Next up and Collection. 

I am at my wits end.  I am a newbie to this website so apologize if I do anything incorrectly or if this is in the wrong page or category. 

The last few weeks, Seren just stops loading Next up. Either it doesnt update it at all and just shows whatever was on it before and nothing new, or now it has recently started showing nothing.  It wipes out everything in the list where I just see dots to go back to the previous menu. When this happens, I have rebooted, reauthorized trakt and real-debrid, rebooted again, nothing helps. I literally today cleared kodi, reinstalled seren from scratch, authorized trakt and Debrid, and next up loaded! Yeah, I just watched one show. I went into collection, then tried to back into next up and I get the dots again!! Really?!?! Why does it keep doing that?! 

As for collection, that hasn't worked in months. It seems to load only 21 videos, per the side bar menu but I have way more then that in my actual trakt account. No matter what I do with reinstalling and reauthorizing everything, the collection hasn't pulled correctly in a very long time. 

Am I doing something wrong? Is there something I should be doing when I install seren, or some setting that will stop the next up from being cleared or the collections from being loaded.  

Again, I apologize if this is not the right place for this or not the right category, I have been trying to find help for these seren specific issues for so long now that it looked like this may be the best place to do that. Thanks for the help!

**Comment by drinfernoo** [comment]
We'll need a log to even *try* and figure this out... but it *sounds* like you need to rebuild your Trakt database.

**Comment by KCRN33** [comment]
I have actually rebuilt the trakt database every time I try to fix these issues, sorry I left that out of the list of things I did. Everytime this happens, sometimes the reinstall of seren or kodi itself makes seren work for a couple of weeks, but lately it seems to only be working a couple days before it needs to be reinstalled and now just an hour. 

You refer to a log. Being that I am using a Nvidia Shield device connected to my tv and talking to you via a website on my iPad, how would I go about getting you any log info you would need to see? Do I need to download any other apps in kodi to get the logs? This most recent reinstall, I cleared everything I had previously installed and just installed seren to see if I could get it to work, it did for the hour, then stopped. So I didn't install anything else yet. Thanks for your advice on how to get this working,  when seren works, it's the best app we use on or off kodi. Would love to get it working again more consistently. Thanks
